### PR TITLE
Update re-indexing table

### PR DESCRIPTION
**Pull request by sbesson**
Following http://lists.openmicroscopy.org.uk/pipermail/ome-users/2015-February/005038.html, the re-indexing times indicated in our docs is out-of-date are largely overestimated.

This PR updates the re-indexing estimated times using:
- data from a background re-indexing of a production database - courtesy of @ehrenfeu 
- data from an off-line re-indexing of a similarly sized production database

Minor nitpicks/corrections are also added to the off-line re-indexing section.

### CODE REVIEW

**Comment by mtbc** [comment]
Staged at https://www.openmicroscopy.org/site/support/omero5.1-staging/sysadmins/search.html.

File: omero/sysadmins/search.txt [hunk 1]
```diff
@@ -199,20 +198,21 @@ servers of various sizes:
...
+    * -Xmx4800m
 
-  - * Background
-    * 10 days
+  - * Off-line [2]_
+    * 6h30
     * 16TB
-    * -Xmx2048M
-    * 100
+    * -Xmx2g
```
Comment (joshmoore) [inline omero/sysadmins/search.txt]: Should these be in terms of `--mem`?

**Comment by joshmoore** [comment]
Other than the one minor comment, I'd say looks good.

File: omero/sysadmins/search.txt (hunk 1 above)
Comment (sbesson) [inline omero/sysadmins/search.txt]: Yes that was the meaning of my `[2]` note. I initially put `--mem=2g` in the table but then thought it didn't render well the above line mentioning the JVM Indexer settings. Any suggestion?

Comment (joshmoore) [inline omero/sysadmins/search.txt]: If it's a pain to have `--mem=2g` in the table, then no worries.

Comment (sbesson) [inline omero/sysadmins/search.txt]: No worries, sir, pushed.

**Comment by hflynn** [comment]
I'm assuming this is good to merge since Josh's only comment is addressed now, thanks.

**Comment by sbesson** [comment]
--rebased-to #1114

**Pull request merged**